### PR TITLE
adding page refresh to unregister test as there is a delay for the cl…

### PR DESCRIPTION
**Pull request by anarang**
…ient machine to understand the unregistration; changing class name

### CODE REVIEW

File: tests/foreman/rhai/test_rhai.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from robottelo.ui.navigator import Navigator
 from robottelo.ui.session import Session
 from robottelo.vm import VirtualMachine
+import time
```
Comment (elyezer) [inline tests/foreman/rhai/test_rhai.py]: We organize imports like the following:

```python
import x
import y
import z

from abc import dfg
from asd import fgh
```

File: tests/foreman/rhai/test_rhai.py [hunk 2]
```diff
@@ -130,6 +131,9 @@ def test_unregister_system_from_rhai(self):
                 session.nav.click(
                     locators['insights.unregister_button']
                 )
+                self.browser.refresh()
+                time.sleep(30)
```
Comment (elyezer) [inline tests/foreman/rhai/test_rhai.py]: Have you tried using `self.wait_for_ajax()`? Using hard sleep is not a good approach and is used only in cases that there are not other options.

**Comment by elyezer** [comment]
ACK, pending comments.

File: tests/foreman/rhai/test_rhai.py (hunk 1 above)
Comment (anarang) [inline tests/foreman/rhai/test_rhai.py]: @elyezer understood, will do that.

File: tests/foreman/rhai/test_rhai.py (hunk 2 above)
Comment (anarang) [inline tests/foreman/rhai/test_rhai.py]: @elyezer I understand that point and tried using all the options, before picking up time.sleep: 
1. self.browser.refresh() only
2. self.browser.refresh and wait_for_ajax()
3.self.browser.refresh() and time.sleep(30)
The first two did not work and thus decided to go with time.sleep()
Here is the output[1] with option 2 where the test fails.
[1] http://pastebin.test.redhat.com/329618

**Comment by sthirugn** [comment]
ACK

File: tests/foreman/rhai/test_rhai.py (hunk 2 above)
Comment (oshtaier) [inline tests/foreman/rhai/test_rhai.py]: As @elyezer mentioned, hard sleep is really bad idea and especially having static amount of seconds. Is it possible verify your conditions dynamically in some way with smaller delay?
Something like:
```
for n tries
    check whether condition is true
    if not  - sleep for let's say 5 seconds
```

**Comment by oshtaier** [comment]
pending question

File: tests/foreman/rhai/test_rhai.py (hunk 2 above)
Comment (anarang) [inline tests/foreman/rhai/test_rhai.py]: @oshtaier I completely understand your concern and thought of the condition thing as well, but then that would be like 'uploading data until we get a status code:412(which is expected)'. I have written my concern to the team but haven't got any updates, so as soon as I get something better for this test to deal with, I shall update it. Till then we could merge it(if you think it is ok to do so) as the test has been failing on Jenkins for a past few builds.

Comment (elyezer) [inline tests/foreman/rhai/test_rhai.py]: @oshtaier @anarang I'll merge this PR and #3065 for tracking improvements.

**Comment by elyezer** [comment]
ACK

**Pull request merged**